website/docs/tutorial/1-setting-up.md

```diff
@@ -74,6 +72,7 @@ select * from final
 ```
 
 ## Choose the way you want to develop
```

Comment:
question: should we move this decision between CLI and Cloud to header of "Create a project"??  That would move it closer to when that decision is relevant?

Reply:
@coapacetic You read my mind! I was going to suggest a place but was not sure where to put it. I love this suggestion and will add it there!

Reply:
I applied these changes here: https://github.com/dbt-labs/docs.getdbt.com/pull/1070/commits/e16053577a9b0a13ca693ca59042172c61bd0c7a